db/Articles.js

```diff
@@ -0,0 +1,81 @@
+//jshint esversion: 6
+
+///NEED TO EXAMINE THIS DB CLOSELY. What needs to be returned to the router in the instantiation?
+class Articles {
+  constructor() {
+    this.title = "";
+    this._collection = [];
+  }
+
+  getAllArticles(){
+    return this._collection;
+  }
+
+  getArticleByTitle(title) {
+    let index = null;
+    this._collection.forEach((object) => {
+      if(object.title === title) {
+        index = this._collection.indexOf(object);
```

Comment:
This is unnecessary since you already have access to the object you want index to point to already. You should be able to just:
``` javascript
  index = object:
```